nucypher/blockchain/eth/agents.py

```diff
@@ -816,23 +844,22 @@ def post_aggregation(
         public_key: DkgPublicKey,
         participant_public_key: SessionStaticKey,
         transacting_power: TransactingPower,
-        fire_and_forget: bool = True,
-    ) -> Union[TxReceipt, HexBytes]:
+    ) -> AsyncTx:
         contract_function: ContractFunction = self.contract.functions.postAggregation(
             ritualId=ritual_id,
             aggregatedTranscript=bytes(aggregated_transcript),
             dkgPublicKey=Ferveo.G1Point.from_dkg_public_key(public_key),
             decryptionRequestStaticKey=bytes(participant_public_key),
         )
-        receipt = self.blockchain.send_transaction(
+        atx = self.blockchain.send_async_transaction(
             contract_function=contract_function,
             gas_estimation_multiplier=1.4,
             transacting_power=transacting_power,
-            fire_and_forget=fire_and_forget,
+            info={"ritual_id": ritual_id, "phase": PHASE2},
```

Comment:
Should't this be a PhaseId? Same comment for the other method.

Reply:
sure, it can be.  The only way this dict is used currently is for async logging.  it has not predefined format other than expecting that every values it string-representable.